### DISCUSSION TITLE
哈尔滨工业大学（本部）

### DISCUSSION THREAD

**Discussion by YanWQ-monad** [CollegesChat/university-information#13] [University]
> 回答来自群友，我仅负责搬运

**宿舍是上床下桌吗？教室和宿舍有没有空调？有独立卫浴吗？没有独立浴室的话，澡堂离宿舍多远？**
宿舍：上限很高，二人单间的独立卫浴套间带空调；下限极低，建于1953年的六人间上床下床无独立卫浴无空调。住到哪要看年级看学院

**有早自习、晚自习吗**
有的话就是部分导员自己整的活，集中在大一刚入学

**有晨跑吗**
无

**每学期跑步打卡的要求是多少公里，可以骑车吗**
40km，可以

**寒暑假放多久，每年小学期有多长**
寒假一个半月多一些，暑假一个月；小学期一个月但是基本没什么课

**学校允许点外卖吗，取外卖的地方离宿舍楼多远**
允许，没有固定外卖点，一般就近栅栏，都很近

**学校附近有地铁站吗**
一校区有（大一在二校区，大二及以后除几个学院外都在一校区）

**宿舍楼有洗衣机吗**
有

**校园网怎么样**
10r/m，30Mbps，一言难尽

**每天断电断网吗，几点开始断**
不断

**食堂价格贵吗，会吃出异物吗**
便宜又好吃

**洗澡热水供应时间**
之前是下午+晚上，最近改成早晚了，随季节变化可能还会改

**校园内可以骑电瓶车吗，电池在哪能充电**
可以，但没必要，校园很小；可能没地方充电

**Comment by yuejiuang** [comment]
目前已经更新，宿舍最差基本上就是那个双人间了（双人上下床），校园网已经到80mbps（wifi）200mbps（有线）部分宿舍无有线。洗澡水全天不断（本人曾凌晨三四点去过，水还是热的）。目前校内已经不能新增电动车，需注意。

**Comment by nanobb** [comment]
经济学怎么样啊 建议文科生去学经济吗….

**Comment by exceedssummon** [comment]
便宜没问题，好吃这块我感觉某一些窗口有点小拉，(一共呆了六年，我去食堂的次数确实比较少…）去的次数最多的是学苑一楼右边的面和馄饨，学苑三楼小是小、自选，和学士一楼最左边的面，学士二楼的自选也还不错(早饭这块样数最多），饺子园去的少，印象中还行。关于面条，来晚了面条浇头凉了，很多家面条就不好吃了。
